nupic/embodied/utils/garage_utils.py

```diff
@@ -232,8 +128,6 @@ def create_dendritic_layer(dendritic_layer):
         return BiasingDendriticLayer
     elif dendritic_layer == "abs_max_gating":
```

Comment:
Would be easier if we get rid of these ifs and pass the class directly when creating the experiment config. That would help in debugging removing this unnecessary layer of indirection.

Reply:
Agreed on this, but I think the main usefulness in keeping the string is so that wandb logs it. 

(In the future, maybe we can ask nupic.research & dendrites to add string IDs to each dendrite layer, and then pass that into wandb instead. Either way, wandb logging everything right now seems like a good option.)